### PR TITLE
[Hotfix] v0.33.1 (Object Storage Guides)

### PR DESCRIPTION
**Pull request by hzoppetti**
Added shortguide for ga note and applied to all - these all had the same or similar note at the top.
Added Frankfurt region to How to Use Object Storage guide.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit bb6903f7efe0f391f2e5dd004e9576bb782d8359

https://deploy-preview-3133--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/object-storage/object-storage-ga-shortguide/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: Heather Zoppetti
+published: 2020-02-12
+title: Object Storage General Availability Notice
+keywords: []
+headless: true
+show_on_rss_feed: false
+---
+
+{{< note >}}
+[Linode Object Storage](/docs/platform/object-storage/) is now available to the general public in the Newark and Frankfurt data centers! Starting November 1, 2019, all customers with the Object Storage service enabled on their account will be billed. For more information, see our [Object Storage Pricing and Limitations](/docs/platform/object-storage/pricing-and-limitations/) guide.
```
Comment (nmelehan) [inline docs/platform/object-storage/object-storage-ga-shortguide/index.md]: hey @hzoppetti, what's your opinion on getting rid of `Starting November 1, 2019, all customers with the Object Storage service enabled on their account will be billed.` in this? People have been charged for a few billing cycles now, so it feels like that's probably the expectation now

Comment (hzoppetti) [reply docs/platform/object-storage/object-storage-ga-shortguide/index.md]: Sounds good to me

**Comment by nmelehan** [comment]
Putting do-not-merge label on till Friday

**Comment by leslitagordita** [comment]
**NOTE**: This will be **hotfix v0.33.1** (disregard branch name).

**Comment by nmelehan** [comment]
@andystevensname 👍 on the new tables

**Pull request merged**